detox/src/devices/android/ADB.test.js

```diff
@@ -55,5 +55,10 @@ xdescribe('ADB', () => {
     await adb.unlockScreen('deviceId');
     expect(exec).toHaveBeenCalledTimes(1);
   });
+
```

Comment:
I guess we should consider dropping unit tests on ADB as well, but for now, don't delete this test